### DISCUSSION TITLE
JASPAR2022 getMatrixSet difference

### DISCUSSION THREAD

**Discussion by Jimmyyun** [stuart-lab/signac#1594] [Q&A] [answered]
Hi Tim,

Hope you are doing well in Singapore!
I've been working with Phospho-seq from John, and looking at motif enrichment using ATAC peaks. I noticed Signac and JASPAR2022 bioconductor have a different way of getting MatrixSet. 

For example, Signac is like
`pfm<-getMatrixSet(x=JASPAR2022,opts=list(collection="CORE",tax_group='vertebrates',all_versions=FALSE))`

but, JASPAR2022 bioconductor is like
`pfm<-getMatrixSet(x=JASPAR2022,opts=list(species=9606,type="SELEX",all_versions=TRUE))`

I guess the biggest difference is whether all versions are TRUE or not, and the type. Do you know the difference and what makes a better approach?

**Comment by timoast** [comment, marked as answer]
Hey Jimmy!

In the Signac vignette we recommend getting the core vertebrate collection which is a more curated, non-redundant set from a variety of sources including SELEX, ChIP-seq, DAP-seq, and others. For scATAC-seq analysis we typically want just one version for each TF.

I think the documentation for JASPAR2022 you're referring to ([here](https://bioconductor.org/packages/release/data/annotation/vignettes/JASPAR2022/inst/doc/JASPAR2022.html)) is just meant as an example of how to query the database, rather than a specific recommendation of what motifs to use in the analysis. For most things they recommend using the core collection: https://jaspar.elixir.no/docs/
